### PR TITLE
[DDW-649] UI - Stake pool list

### PR DESCRIPTION
**Pull request by daniloprates**
This PR implements the Stake pool list.

## Todos

- [x] Stake pools list
- [x] Stake pool colors
- [X] Basic Tooltip
- [X] Remove Stake pool link at Rewards page

- **Search and filters (All | New | Charity) are not in the scope of this PR**
- **The tooltip positioning and closing on ESC will be handled on a different card. For now, the tooltip will be hidden if the pool is close to the edge of the page.**

## Screenshots

image

image

---

## Testing Checklist

- _Slack QA thread_
- [ ] Test Storybook story under: `Staking > Stake Pools`

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/components/staking/stake-pools/StakePool.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+.index {
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+}
+
+.clock {
+  background: var(--theme-staking-stake-pool-retirement-background-color);
+  width: 18px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePool.scss]: @daniloprates please order CSS rules alphabetically 🙏

File: source/renderer/app/components/staking/stake-pools/StakePool.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  top: 0;
+  right: 0;
+}
+
+.clockIcon {
+  display: block;
+  margin: 0;
+  padding: 0;
+  line-height: 0px;
+  text-align: center;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePool.scss]: @daniloprates please order CSS rules alphabetically 🙏

File: source/renderer/app/components/staking/stake-pools/StakePool.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    height: 15px;
+  }
+}
+
+.colorBand {
+  display: block;
+  width: 100%;
+  height: 5px;
+  position: absolute;
+  bottom: 0px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePool.scss]: @daniloprates please order CSS rules alphabetically 🙏

File: source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  border-radius: 5px;
+  border: 1px solid var(--theme-staking-stake-pool-tooltip-border-color);
+  box-shadow: 0 5px 20px 0 var(--theme-staking-stake-pool-tooltip-shadow-color);
+  display: none;
+  left: 50px;
+  margin: 20px;
+  padding: 20px;
+  position: absolute;
+  top: -60px;
+  width: 250px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss]: @daniloprates please order CSS rules alphabetically 🙏

**Comment by daniloprates** [comment]
@nikolaglumac I thought of doing the tooltip part in a different PR, as it's a completely different context and logic. It'd be good to have this PR merged ASAP, to avoid merging hell from other Staking cards.

File: source/renderer/app/components/staking/stake-pools/StakePool.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  right: 0;
+  top: 0;
+  width: 18px;
+}
+
+.clockIcon {
+  display: block;
+  line-height: 0px;
+  margin: 0;
+  padding-top: 1px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePool.scss]: @daniloprates here you set top padding to 1px and then in the following line back to zero - is that on purpose?

File: source/renderer/app/components/staking/stake-pools/StakePool.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  padding: 0;
+  text-align: center;
+
+  svg {
+    width: 15px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePool.scss]: @daniloprates please sort these CSS rules alphabetically too 🙏

File: source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+.retirement {
+  span {
+    background: var(
+      --theme-staking-stake-pool-tooltip-table-param-retirement-background-color
+    );
+  }
+}
+
+.delegateButton {
+  width: 100% !important;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss]: @daniloprates please sort these CSS rules alphabetically too 🙏

**Comment by nikolaglumac** [comment]
> @nikolaglumac I thought of doing the tooltip part in a different PR, as it's a completely different context and logic. It'd be good to have this PR merged ASAP, to avoid merging hell from other Staking cards.
OK

File: Desktop/SingleTooltip.scss [hunk 8]
```diff
@@ -0,0 +1,9 @@
+.component {
```
Comment (nikolaglumac) [inline Desktop/SingleTooltip.scss]: @daniloprates why do we have this `Desktop/SingleTooltip.scss` file?

Comment (daniloprates) [reply Desktop/SingleTooltip.scss]: I meant to move it to my Desktop, not to create a directory 😅

**Pull request merged**